blockchain/chaingen/generator.go

```diff
@@ -534,8 +535,8 @@ func isTicketPurchaseTx(tx *wire.MsgTx) bool {
 		return false
 	}
 	txOut := tx.TxOut[0]
-	scriptClass := txscript.GetScriptClass(txOut.Version, txOut.PkScript)
-	return scriptClass == txscript.StakeSubmissionTy
+	return standalone.IsStakeOpCodeScript(txOut.Version,
```

Comment:
This is not consensus code.  It should not be updated.  Especially since a big goal of the generator is generally to avoid using consensus code to test consensus code since then you aren't actually testing anything more than `if x == x`.